validator/accounts/accounts_activation.go

```diff
@@ -0,0 +1,270 @@
+package accounts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/promptutil"
+	"github.com/prysmaticlabs/prysm/validator/accounts/iface"
+	"github.com/prysmaticlabs/prysm/validator/accounts/prompt"
+	"github.com/prysmaticlabs/prysm/validator/accounts/wallet"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	"github.com/prysmaticlabs/prysm/validator/keymanager"
+	"github.com/prysmaticlabs/prysm/validator/keymanager/imported"
+	"github.com/urfave/cli/v2"
+)
+
+func DisableAccountCli(cliCtx *cli.Context) error {
+	w, err := wallet.OpenWalletOrElseCli(cliCtx, func(cliCtx *cli.Context) (*wallet.Wallet, error) {
```

Comment:
I saw this similar code across multiple files in the account directory (with slight modification related to flags, prompt, custom error messages).

Just wondering if it's "on purpose" and considered best practice for Prysm instead of DRYing it?

Reply:
Totally agreed this is repetitive. The subtle differences are in the callback function to `OpenWalletOrElseCli`. Perhaps we can further consolidate it? Maybe the open wallet or else cli can also return the keymanager?